packages/lab/src/stepped-tracker/Step.Label.tsx

```diff
@@ -0,0 +1,54 @@
+import { Text, type TextProps, makePrefixer } from "@salt-ds/core";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { useWindow } from "@salt-ds/window";
+import clsx from "clsx";
+
+import type { Step } from "./Step";
+import stepLabelCSS from "./Step.Label.css";
+
+export namespace StepLabel {
+  export interface Props extends TextProps<"div"> {
+    id: string;
+    stage: Step.Stage;
+    status?: Step.Status;
+  }
+}
+
+const withBaseName = makePrefixer("saltStepLabel");
+
+export function StepLabel({
+  id,
+  stage,
+  status,
+  className,
+  styleAs = "label",
+  children,
+  ...props
+}: StepLabel.Props) {
+  const targetWindow = useWindow();
+
+  useComponentCssInjection({
+    testId: "salt-step-label",
+    css: stepLabelCSS,
+    window: targetWindow,
+  });
+
+  const elucidation: Step.Elucidation = status || stage;
```

Comment:
Where does this word come from? New language needs design and dev alignment

Reply:
It's internal use only, I wanted to strongly type the clarrification message (elucidation) that get's read to the user for a11y purposes. I picked elucidation because it's a very uncommon word, and variant already existed (deprecated now).

Reply:
removed it